contracts/zero-ex/contracts/src/migrations/FullMigration.sol

```diff
@@ -62,25 +66,39 @@ contract FullMigration {
         _initialMigration = new InitialMigration(address(this));
     }
 
+    /// @dev Retrieve the bootstrapper address to use when constructing `ZeroEx`.
+    /// @return bootstrapper The bootstrapper address.
+    function getBootstrapper()
+        external
+        view
+        returns (address bootstrapper)
+    {
+        return address(_initialMigration);
+    }
+
     /// @dev Deploy the `ZeroEx` contract with the full feature set,
     ///      transfer ownership to `owner`, then self-destruct.
     /// @param owner The owner of the contract.
+    /// @param zeroEx The instance of the ZeroEx contract. ZeroEx should
+    ///        been constructed with this contract as the bootstrapper.
     /// @param features Features to add to the proxy.
-    /// @return zeroEx The deployed and configured `ZeroEx` contract.
+    /// @return _zeroEx The configured ZeroEx contract. Same as the `zeroEx` parameter.
     /// @param migrateOpts Parameters needed to initialize features.
     function deploy(
         address payable owner,
+        ZeroEx zeroEx,
         Features memory features,
         MigrateOpts memory migrateOpts
     )
         public
-        returns (ZeroEx zeroEx)
+        returns (ZeroEx _zeroEx)
```

Comment:
Not really other than something I noticed during migrations is it's nice to have return values so you can feel confident that an eth_call succeeded. With old versions of geth it wasn't clear whether a function with no result  reverted or succeeded because both return a `0x`.